src/vimeo/players/VimeoDesktopArticleVideoPlayer.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useRef, useState } from 'react';
+import { VimeoArticleVideoPlayerProps } from 'vimeo/types';
+import useOnScreen from 'utils/useOnScreen';
+import PlayerWrapper from 'vimeo/players/shared/PlayerWrapper';
+import WDSVariables from '@fandom-frontend/design-system/dist/variables.json';
+import CloseButton from 'vimeo/players/shared/CloseButton';
+import VimeoPlayerWrapper from 'vimeo/players/shared/VimeoPlayerWrapper';
+import styled, { css, keyframes } from 'styled-components';
+
+const VimeoDesktopArticleVideoTopPlaceholder = styled.div`
+	position: absolute;
+	width: 100%;
+	padding-top: 56.25%;
+	top: 0;
+	left: 0;
+	bottom: 0;
+	right: 0;
+	z-index: 2;
```

Comment:
I wonder, why not use `WDSVariables` for that `z-index` value?

Reply:
That's a good question ;-) It was "heavily inspired" by Youtube Takeover, so I'll try out using the WDSVariables 👍